src/admin-pages/Dashboard/index.tsx

```diff
@@ -3,7 +3,7 @@ import { Layout, Menu, Breadcrumb, Card, Row, Col } from 'antd'
 
 const Dashboard = () => {
   return (
-    <div className={style.dashboardContainer}>
+    <div className={style.dashboardContainer} style={{ backgroundColor: 'red' }}>
```

Comment:
why ?

Reply:
just for testing, I remember that I have delete it, weird.